src/graph/executor/admin/SubmitJobExecutor.cpp

```diff
@@ -151,7 +151,7 @@ nebula::DataSet SubmitJobExecutor::buildShowResultData(
                           apache::thrift::util::enumNameSafe(tsk.get_result()),
                           convertJobTimestampToDateTime(std::move(tsk).get_start_time()),
                           convertJobTimestampToDateTime(std::move(tsk).get_stop_time()),
-                          jd.get_code() == nebula::cpp2::ErrorCode::E_UNKNOWN
+                          jd.get_code() == nebula::cpp2::ErrorCode::E_JOB_PROCESSING
```

Comment:
Remember to add `E_JOB_PROCESSING` to common.thrift and `MetaClient::handleResponse`

Reply:
Thanks.